templates/changepass/resetPass.html

```diff
@@ -0,0 +1,67 @@
+{% load static %}
+
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <!-- Load bootstrap css, jquery, bootstrap js -->
+    <!-- common across all applications inheriting from dashboard -->
+    <link href="{% static 'css/bootstrap.min.css' %}" rel="stylesheet">
+    {% if user_type and user_type == constant_ut_adminStaff %}
+    <link rel="stylesheet" type="text/css" href="{% static 'css/admin_theme.min.css' %}">
+{% else %}
+    <link rel="stylesheet" type="text/css" href="{% static 'css/student_theme.min.css' %}">
+{% endif %}
+<link href="{% static 'changepass/css/changepass.css' %} " rel="stylesheet">
+    <script src="{% static 'js/jquery-3.3.1.min.js' %}"></script>
+    <script src="{% static 'js/popper.js' %}"></script>
+    <script src="{% static 'js/bootstrap.min.js' %}"></script>
+    <script src="https://kit.fontawesome.com/fc24e2383c.js" crossorigin="anonymous"></script>
+    <!-- Custom styles for this template -->
+    <title>oneapply</title>
```

Comment:
Having this as OneApply would be consistent

Reply:
done